osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/datetime/ActivityCalendarValidatorTest.java

```diff
@@ -186,6 +191,20 @@ void testMultipleUsesOfSameWeek() {
                 this.createActivityCalendarDto(TestAnomaly.MULTIPLE_USES_SAME_WEEK)));
   }
 
+  @Test
+  void testInvalidSeasonName() {
+    final byte[] test = new byte[] {"0".getBytes()[0]};
```

Comment:
How does variable _test_ contribute in this test?

Reply:
It's some testcode I forgot to remove.